src/components/TextField/TextField.tsx

```diff
@@ -41,7 +41,7 @@ export const TextField: React.FC<Props> = ({
           id={id}
           data-cy={`movie-${name}`}
           className={classNames('input', {
-            'is-danger': hasError,
+            'is-danger': value === '' && hasError,
```

Comment:
'Value is not empty' is already checked in hasError variable in line 30 (!value)
```suggestion
            'is-danger': hasError,
```